documentation-website/Writerside/snippets/exposed-dao/src/main/kotlin/org/example/examples/DeleteExamples.kt

```diff
@@ -0,0 +1,13 @@
+package org.example.examples
+
+import org.example.entities.StarWarsFilmEntity
+
+class DeleteExamples {
+    fun deleteFilm() {
+        val movie = StarWarsFilmEntity.findById(2)
+        if (movie != null) {
+            val deletedMovie = movie.delete()
+            println(deletedMovie)
```

Comment:
I can do another review whenever you're ready to request a new one, but I saw this when troubleshooting.

`Entity.delete()` doesn't return a value , but using line 9 `val deletedMovie = movie.delete()` gives the impression that it might return the entity or something.
This might be a case to use individual lines in code block, or change the entire function:
```kt
fun deleteFilm() {
    val movie = StarWarsWFilmEntity.findById(2)
    movie?.delete()
}
```

Reply:
Thank you! I've fixed the examples as suggested and updated the code refs.